modules/metering-install-verify.adoc

```diff
@@ -9,7 +9,7 @@ To verify your metering installation you can check that all the required Pods ha
 
 .Procedure
 
-. Navigate to *Workloads* -> *Pods* in the metering namespace and verify that Pods are being created. This can take up to a minute after installing the metering stack.
+. Navigate to *Workloads* -> *Pods* in the metering namespace and verify that Pods are being created. This can take up several minutes after installing the metering stack.
 +
 You can run the same check using kubectl:
```

Comment:
@lamek should all references to `kubectl` be converted to `oc` if this is a Openshift document?